crates/ironrdp-tls/src/lib.rs

```diff
@@ -26,20 +26,9 @@ compile_error!("a TLS backend must be selected by enabling a single feature out
 pub use impl_::{upgrade, TlsStream};
 
 #[cfg(any(feature = "native-tls", feature = "rustls"))]
```

Comment:
Also remove this line, it was there just to suppress the dead code warning when no backend was enabled, but now it’s part of the public API